app/pages/patient/patientinfo.js

```diff
@@ -298,9 +297,10 @@ var PatientInfo = React.createClass({
     } else {
       return 'Birthdate not known';
```

Comment:
One thing I have just been triangulating around is the behaviour of dateDifference in sundial (but really the behaviour of diff in moment.js). 

If you take a look at the [documentation](http://momentjs.com/docs/#/displaying/difference/), there i is an edge case for using diff in this regard when looking at at difference between dates within 1 year. The point is best illustrated with a [jsFiddle](http://jsfiddle.net/ho70pxn4/2/) I have created.

Basically, when a diff value is between -1 and 0, using date1.diff(date2, 'years') in this range rounds the value up to 0. Whereas, when returning a floating point, by setting the optional third value to true, you can see that indeed the underlying value is negative.

In relation to this request, it can cause an oddity here where if a birthdate is in the future, but within 365 days of the current date, the text rendered would be 'Born this year'/'Diagnosed this year'

For the time being I'll leave this edge case in, perhaps in the future we can address this by allowing a  4th argument into dateDifference to allow floating point values to be returned.